PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/AdjustImageWindow.xaml.cs

```diff
@@ -256,5 +260,37 @@ private void ZoomOutButton_OnClick(object sender, RoutedEventArgs e)
         {
             _croppingAdorner.ZoomCroppingRect(-AdjustUnit);
         }
+
+        private void FilpOrRotate(RotateFlipType roatateFlipType)
+        {
+            var img = (Bitmap)Bitmap.FromFile(thumbnailImageFile.Text);
```

Comment:
following SaveCropButton_OnClick(),
so use `CropResult` instead of `thumbnailImageFile.Text `